### DISCUSSION TITLE
Quantum Communications Constellations Hypervolume wrong reference point

### DISCUSSION THREAD

**Discussion by Cosijopiii** [esa/SpOC2#9] [Quantum Constellations Q&A] [answered]
Hello! 

 The quality of your approximated Pareto frontier will then be assessed by the hypervolume indicator with a reference point of 
 [1.4,1.2] 

When I use the reference point that you mention([1.4,1.2]), the hypervolume appears different than the one I submit, doing tests, when I change the values to [1.2, 1.4] the Hypervolume coincides with that of the Leaderboard, the value shown is incorrect or it is only backwards ?

**Comment by jazzalin** [comment, marked as answer]
Hi @Cosijopiii, thanks for notifying us!

Indeed the correct reference point is [1.2, 1.4]. I am afraid that the problem description has it backwards and does not reflect the implementation. We will update it shortly.

Best

**Reply by Cosijopiii** [reply]
Thanks you so much. 👌
